README.md

```diff
@@ -1,19 +1,31 @@
-# Stereum 2.0
+![GitHub - Stereum 2 0 - ReadMe](https://user-images.githubusercontent.com/82385103/175423093-03d28d7b-3c2e-44b7-8be1-7f2a5946fcc6.png)
+
+---
 
 [![Support Server](https://img.shields.io/badge/Discord-Stereum-blue?style=flat-square)](https://discord.gg/8Znj8K6GjN) [![Support Email](https://img.shields.io/badge/Email-Stereum-brightgreen?style=flat-square)](mailto:stereum@stereum.net)
 
-This repository contains the code and scripts for ~~[Stereum 2.0](https://stereum.net/ethereum-node/)~~ (tba), an [Ethereum](https://ethereum.org/) node setup and management software with focus on self sovereignity, privacy and flexibility.
+---
+This repository contains the code and scripts for Stereum 2.0 (tba)
```

Comment:
I think we can remove the "(tba)" here, dont you?

Reply:
Agree

Reply:
still in? 🤔